extensions/elasticsearch-rest-client-common/deployment/src/main/java/io/quarkus/elasticsearch/restclient/common/deployment/ElasticsearchDevServicesBuildTimeConfig.java

```diff
@@ -34,9 +34,9 @@ public class ElasticsearchDevServicesBuildTimeConfig {
 
     /**
      * The value for the ES_JAVA_OPTS env variable.
-     * Defaults to setting the heap to 1GB.
+     * Defaults to setting the heap to 512MB min - 1GB max.
      */
-    @ConfigItem(defaultValue = "-Xmx1g")
+    @ConfigItem(defaultValue = "-Xms512m -Xmx1g")
```

Comment:
LGTM, but out of curiosity: is this related to the testcontainers version bump in any way? How?

Reply:
Yes, see https://github.com/quarkusio/quarkus/pull/28388#issuecomment-1268237190